test/test_mission_manager/test_mission_manager.cpp

```diff
@@ -3,27 +3,36 @@
 #include <unity.h>
 void test_valid_initialization()
 {
-    MissionManager mission_manager(0);
```

Comment:
revert all of this if you can but if not no worries- it will all be overwritten by the new version for the new mission modes